### PR TITLE
fix: invalid basepath, feat: ability to select config by name only

### PR DESCRIPTION
**Pull request by debendraoli**
### Commit Message

```bash
type: commit message
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

```bash
version: <log entry>
```

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [ ] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: test/e2e/config.yaml [hunk 1]
```diff
@@ -31,8 +31,8 @@ chains:
       name: archway
       chain_id: archway-1
       image:
-        repository: archwaynetwork/archwayd
-        version: sha-8f53ac8
```
Comment (AntonAndell) [inline test/e2e/config.yaml:35]: Why not use this archway image that is on dockerhub?

Comment (debendraoli) [reply test/e2e/config.yaml:35]: I am getting no such image response. are you able to pull?

Comment (AntonAndell) [reply test/e2e/config.yaml:35]: Yeah works for me
https://hub.docker.com/layers/archwaynetwork/archwayd/sha-8f53ac8/images/sha256-533fa41e3e39c9b4af1a49baffd6e7db782c2e65da9be7b1d105f2f6f643475d?context=explore

Comment (redlarva) [reply test/e2e/config.yaml:35]: I guess, https://hub.docker.com/r/archwaynetwork/archwayd is official docker image

Comment (debendraoli) [reply test/e2e/config.yaml:35]: Oh I see, its platform related then.
I can pull using amd64 but not arm64.

**Review by redlarva — APPROVED** [review]
looks good

**Pull request merged**